.changeset/kind-beers-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hashicorp/react-card': patch
+---
+
+Removes duplicate props docs
```

Comment:
I don't think a release is needed here to just update docs.

Reply:
Hmm, true :+1:

Reply:
🔪 
https://github.com/hashicorp/react-components/pull/818/commits/bbaabb4136daf2c18e0c9f0a19a1c542e56cd1b0